install.sh

```diff
@@ -28,6 +28,12 @@ eval "xcode-select --install"
 printf "\n${YELLOW}Installing Homebrew...${NORMAL}\n"
 eval '/usr/bin/ruby -e "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/master/install)"'
 
+# If this is an M1 mac
+if [[ `uname -m` == 'arm64' ]]; then
+  echo 'eval "$(/opt/homebrew/bin/brew shellenv)"' >> "/Users/$USER/.zprofile"
```

Comment:
I'm 99% sure that interpolating `$USER` here will work

Reply:
Confirmed! As does `~/.zprofile` fwiw